From 8a246230bd2e822f440ebf4e7424fabb4d42da24 Mon Sep 17 00:00:00 2001
From: Fathi Boudra <fathi.boudra@linaro.org>
Date: Fri, 31 Mar 2017 11:41:30 +0300
Subject: [PATCH] selftests: gpio: add config fragment for gpio-mockup

The gpio test requires to insert the gpio mockup module (CONFIG_GPIO_MOCKUP).
The gpio mockup driver depends on gpiolib (CONFIG_GPIOLIB).
CONFIG_GPIO_SYSFS is selected automatically by the gpio mockup driver.

Tested on x86_64 and arm64 with defconfig and kselftest-merge.

Signed-off-by: Fathi Boudra <fathi.boudra@linaro.org>
Signed-off-by: Shuah Khan <shuahkh@osg.samsung.com>
---
 tools/testing/selftests/gpio/config | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100644 tools/testing/selftests/gpio/config

diff --git a/tools/testing/selftests/gpio/config b/tools/testing/selftests/gpio/config
new file mode 100644
index 000000000000..abaa6902b7b6
--- /dev/null
+++ b/tools/testing/selftests/gpio/config
@@ -0,0 +1,2 @@
+CONFIG_GPIOLIB=y
+CONFIG_GPIO_MOCKUP=m
-- 
2.14.1

